Description:
 Disable close_output, it is flaky.
 .
 It is already fixed upstream in #8587 and #8668, however these depend on
 subsequent changes to the same file and is hard to backport. So just disable
 it for now in Debian, it is a test problem not a cargo problem.
 .
 FIXME remove this patch in cargo 0.49 which contains those above FRs.
--- a/tests/testsuite/build.rs
+++ b/tests/testsuite/build.rs
@@ -4924,8 +4924,8 @@
     p.process(&p.bin("foo")).run();
 }
 
-#[cargo_test]
-fn close_output() {
+#[allow(dead_code)]
+fn _close_output() {
     // What happens when stdout or stderr is closed during a build.
 
     // Server to know when rustc has spawned.
